### ISSUE TITLE
Couldn't find out navigation drawer feature in the app.

### ISSUE THREAD

**Issue by praveenbbhati** [nic-delhi/AarogyaSetu_Android#194]
The code is written in the codebase for showing navigation drawer, but we really couldn't find out the navigation drawer in the app.

Does anyone observe the same?

**Comment by iamareebjamal** [comment]
https://github.com/nic-delhi/AarogyaSetu_Android/blob/89214652eb93ecae355dc7786f1aba4c2c06e843/app/src/main/java/nic/goi/aarogyasetu/views/HomeActivity.java#L261-L293

https://github.com/nic-delhi/AarogyaSetu_Android/blob/master/app/src/main/java/nic/goi/aarogyasetu/views/HomeNavigationView.kt

**Comment by Vanshbajaj** [comment]
yes

**Comment by pikvik** [comment]
Its in the omeActivity.java the snippet:  
setupNavigationMenu() { 
     final DrawerLayout drawerLayout = findViewById(R.id.drawer); 
  
     // Hide and show hamburger menu and menu action 
     menuIntro = findViewById(R.id.hamburger_menu_intro); 
     int menuIntroCount = Integer.parseInt(SharedPref.getStringParams(this, SharedPrefsConstants.MENU_INTRO_COUNT, SharedPrefsConstants.DEFAULT_MENU_INTRO_COUNT)) + 1; 
     if (menuIntroCount > Constants.MAX_INTRO_VIEWS) { 
         menuIntro.setVisibility(Vie

**Comment by praveenbbhati** [comment]
@iamareebjamal Yes, code is there. But we are not able to see this feature in the application.

Look at this photo. and I'm already logged in. 

![IMG_20200527_162117_2](https://user-images.githubusercontent.com/10512037/83022569-58238100-a049-11ea-82db-079b99bd9f2e.jpg)
